tdrs-backend/tdpservice/data_files/views.py

```diff
@@ -147,9 +147,89 @@ def download(self, request, pk=None):
     def download_error_report(self, request, pk=None):
         """Generate and return the parsing error report xlsx."""
         datafile = self.get_object()
-        parser_errors = ParserError.objects.all().filter(file=datafile)
-        serializer = ParsingErrorSerializer(parser_errors, many=True, context=self.get_serializer_context())
-        return Response(get_xls_serialized_file(serializer.data))
+        all_errors = ParserError.objects.filter(file=datafile)
+        filtered_errors = None
+        user = self.request.user
+        is_active = "Active" in datafile.section
+        is_closed = "Closed" in datafile.section
+
+        # We only filter Active and Closed submissions. Aggregate and Stratum return all errors.
+        if not (user.is_ofa_sys_admin or user.is_ofa_admin) and (is_active or is_closed):
```

Comment:
not directly related to your PR, but If we are intending to use `is_ofa_sys_admin` or `user.is_ofa_admin` together, it is better to cache these properties since these will be two queries